setup.py

```diff
@@ -13,6 +13,26 @@ def readme():
     with open('README.rst') as f:
         return f.read()
 
+docs_require = [
+    'sphinx>=2.1',
+    'sphinx-autodoc-typehints',
+    'sphinx_rtd_theme',
+    'nbsphinx',
+    'pandoc',
+    'ipython'
```

Comment:
I think that Jupyter will install ipython

Reply:
Huh, In fact both `pandoc` and `ipython` seem to be (indirect) `Jupyter` dependencies. 
Let's give it a try!